### PR TITLE
Fix bitcoin-tx utilities test.

### PR DESCRIPTION
**Pull request by onokatio**
Fix about test/util/bitcoin-util-test.py

### CODE REVIEW

**Comment by onokatio** [comment]
# Note: Address generate
(to fix src/test/base58_tests.cpp)

- Litecoin's change: https://github.com/litecoin-project/litecoin/commit/07bf9007d31969290b19934541b1fd07b150a784#diff-4e357c8065b1586429b4e6b885ed8e4d
- Bitcoin address prefix: https://en.bitcoin.it/wiki/List_of_address_prefixes
- Base85Check Encode: https://en.bitcoin.it/wiki/Base58Check_encoding
- Address generateing step: https://en.bitcoin.it/wiki/Technical_background_of_version_1_Bitcoin_addresses
- Base58Check web encode&decode tool: http://lenschulwitz.com/base58

File: test/util/data/bitcoin-util-test.json [hunk 1]
```diff
@@ -1,114 +1,114 @@
 [
-  { "exec": "./bitcoin-tx",
+  { "exec": "./monacoin-tx",
```
Comment (ghost) [inline test/util/data/bitcoin-util-test.json]: WTF

Comment (onokatio) [reply test/util/data/bitcoin-util-test.json]: !
Sorry, I mistaked.

**Comment by onokatio** [comment]
# Note: Transaction Version

```
2018-01-19 21:38:29,421 - ERROR - Output data mismatch for blanktxv2.json (format json)
2018-01-19 21:38:29,422 - ERROR - Output formatting mismatch for blanktxv2.json:
*** blanktxv2.json
--- returned
***************
*** 1,7 ****
  {
!     "txid": "4ebd325a4b394cff8c57e8317ccf5a8d0e2bdf1b8526f8aad6c8e43d8240621a",
!     "hash": "4ebd325a4b394cff8c57e8317ccf5a8d0e2bdf1b8526f8aad6c8e43d8240621a",
!     "version": 2,
      "size": 10,
      "vsize": 10,
      "locktime": 0,
--- 1,7 ----
  {
!     "txid": "d21633ba23f70118185227be58a63527675641ad37967e2aa461559f577aec43",
!     "hash": "d21633ba23f70118185227be58a63527675641ad37967e2aa461559f577aec43",
!     "version": 1,
      "size": 10,
      "vsize": 10,
      "locktime": 0,
```

An error has occurred because of diffirent transaction version.
The version is define ad [BIP68](https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki). 
This BIP allows blockchain to use HTLC(Hashed Timelock Contracts).
**It is soft fork.**

I think it is need for bitzeny, but it is **softfork**.
So i posted about it to forum. (https://bitzeny.info/d/39-z2-0-x/297)
Please read. I want your opinion.

**Comment by onokatio** [comment]
rebaseしました

**Comment by onokatio** [comment]
@rmdfwqclmrkcvpbe @hitobb 
Please review.

File: test/util/data/bitcoin-util-test.json [hunk 2]
```diff
@@ -1,274 +1,226 @@
...
+  { "exec": "./bitzeny-tx",
     "args": ["-json","-create", "nversion=1"],
     "output_cmp": "blanktxv1.json",
     "description": "Creates a blank v1 transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
-    "args": ["-"],
-    "input": "blanktxv2.hex",
-    "output_cmp": "blanktxv2.hex",
-    "description": "Creates a blank transaction when nothing is piped into bitcoin-tx"
```
Comment (ghost) [inline test/util/data/bitcoin-util-test.json:16]: Why do you remove tests?

Comment (onokatio) [reply test/util/data/bitcoin-util-test.json]: I forgot about it. so I mistaked.
I will add those test fix soon.

Comment (onokatio) [reply test/util/data/bitcoin-util-test.json:16]: Because it tests about version2 transaction.

**Comment by onokatio** [comment]
I add some test.

**Comment by onokatio** [comment]
@rmdfwqclmrkcvpbe @hitobb
Please review.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**